client/src/components/Header/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Logo from '../../assets/icons/HeaderLogo.png';
+import {Appbar, Avatar, Provider as PaperProvider} from 'react-native-paper';
+import {View} from 'react-native';
+import styles from './style';
+
+function Header() {
+  return (
+    <PaperProvider>
```

Comment:
You don't need `<PaperProvider>` here because I already put it in the main index.js file as a provider for the whole project, so you can just edit this `import {Appbar, Avatar, Provider as PaperProvider} from 'react-native-paper';
` to `import {Appbar, Avatar} from 'react-native-paper';` and remover the provider from the header function.